spacy/pipeline/edit_tree_lemmatizer.py

```diff
@@ -150,7 +149,7 @@ def predict(self, docs: Iterable[Doc]) -> List[Ints2d]:
             # Handle cases where there are no tokens in any docs.
             n_labels = len(self.cfg["labels"])
             guesses: List[Ints2d] = [
-                self.model.ops.alloc((0, n_labels), dtype="i") for doc in docs
+                self.model.ops.alloc2i(0, n_labels, dtype="i") for _ in docs
```

Comment:
```suggestion
                self.model.ops.alloc2i(0, n_labels) for _ in docs
```

Reply:
There is no harm in changing this here, but I think we should revert the source of this error in the next patch release in thinc and postpone it to v8.2 or v9.

Reply:
Sure - let's look at that as a separate task!

Reply:
But I wanted this edit in this PR. There's no need to have the type twice.

Reply:
Sorry, the thread got confusing with the two different topics and I forgot to actually merge the recommendation.

Reply:
https://github.com/explosion/spaCy/pull/11615